demos/digit-prediction/digits_prediction.nlogo

```diff
@@ -0,0 +1,648 @@
+extensions [ py ]
+
+globals [
+  coords-x coords-y color-list digit
+  zero one two three four five six seven eight nine
+]
+
+to setup-py
+  clear-all
+  py:setup py:python
+  (py:run
+    "import numpy as np"
+    "from keras.models import load_model"
+  )
+  ;; load model
+  py:run "model = load_model(\"model.h5\")"
+
+  ;; create coords list, useful to make color_list
+  set coords-x (range max-pxcor (min-pxcor - 1) -1)
```

Comment:
`coords-x` still needs to be `(range min-pxcor (max-pxcor + 1))`. This is because is column indices have the same directionality as x cartesian coordinates, but row indices are have opposite directionality as y cartesian coordinates. You'll notice, right now, 5s and 2s are confused, 7s are generally not recognized, and so forth. This will fix that.